src/helpers/repoutils.ts

```diff
@@ -15,15 +30,15 @@ export class RepoUtils {
 
     //Checks to ensure it's a Team Foundation Git repo, then ensures it's hosted on visualstudio.com
     public static IsTeamFoundationServicesRepo(url: string): boolean {
-        if (RepoUtils.IsTeamFoundationGitRepo(url) && url.toLowerCase().indexOf(".visualstudio.com") >= 0) {
+        if (url.toLowerCase().indexOf(".visualstudio.com") >= 0) {
             return true;
         }
         return false;
     }
 
-    //Checks to ensure it's a Team Foundation repo, then ensures it's *not* on visualstudio.com
+    //This is the "catch all" method.  A repository hosted on TFS is defined by not being on "visualstudio.com"
     public static IsTeamFoundationServerRepo(url: string): boolean {
-        if (RepoUtils.IsTeamFoundationGitRepo(url) && !RepoUtils.IsTeamFoundationServicesRepo(url)) {
+        if (!RepoUtils.IsTeamFoundationServicesRepo(url)) {
```

Comment:
This doesn't seem correct anymore. Can we really say that an URL is an OnPrem url just because it isn't a VSTS URL? Perhaps we should rename the method to "IsPossiblyTeamFoundationServerRepo". But I may be missing something. I know you check the URL by calling the server, so this may not matter.